web-app/src/components/BookingModal/ModalStatusBanner/index.jsx

```diff
@@ -23,11 +23,14 @@ class ModalStatusBanner extends Component {
     }
   };
 
+  toggleConversationMessages = () => { };
```

Comment:
Is this needed?

Reply:
This is the function I am fulfilling for additional legacy message conversation rendering following the new features from today's demo